source/intro.md

```diff
@@ -871,9 +948,9 @@ Bar plot of the ten Aboriginal languages most often reported by Canadian residen
 There are many Python functions in the `pandas` package (and beyond!), and 
 nobody can be expected to remember what every one of them does
 or all of the arguments we have to give them. Fortunately, Python provides 
-the `help` function and `__doc__` attrribute, which 
+the `help` function, which 
 provides an easy way to pull up the documentation for 
-most functions quickly. To use the `help` function to access documentation, you 
+most functions quickly. To use the `help` function to access the documentation, you 
 just put the name of the function you are curious about as an argument inside the `help` function.
 For example, if you had forgotten what the `pd.read_csv` function
```

Comment:
I'm not against it, but I've never got either shift+tab or ctrl+I (or just right click+contextual help) to work reliably. Both most often just do nothing when I try to activate them....

Are you able to get it to work in our course container? Maybe we missed installing something in our docker container?

The nice thing about `help()` is that it works regardless of whether you're using Jupyter (Lab)

Reply:
Yup it works fine in the course container and I was showing it to my students this year. One thing that sometimes make it seem like it is not working is that for R you have to have the cursor positioned on the function name just before the open parenthesis:

![image](https://user-images.githubusercontent.com/4560057/208449912-1f44cc9d-7528-43cb-84c9-b6d801294c27.png)

![image](https://user-images.githubusercontent.com/4560057/208450014-d626e41b-20ce-449a-8c60-796aaee2dee5.png)

Ctrl + I / contextual help also works fine for me for both languages, I think it can be good to show this too. I tend to recommend novices to always have it open to learn as they start out. I agree it is still nice to mention `help` since it works everyone, but I don't think it is as convenient to use in the notebook.

Reply:
@joelostblom Can you verify that it works in our python container please? That's where I tested it last night and it wasn't working.

(the image is listed in the `build_html.sh` script)

And then if it does, please edit the chapter directly to incorporate this info. I've added a little bold text where you should insert the new text if needed. 

If not, just delete that bolded bit please.

Reply:
Ok I will look into that but I don't think that needs to hold up this PR. I opened a follow up issuehttps://github.com/UBC-DSCI/introduction-to-datascience-python/issues/69